proto/bitsong/auction/v1beta1/auction.proto

```diff
@@ -12,13 +12,24 @@ message Auction {
     option (gogoproto.equal) = true;
 
     uint64 id = 1;
-    uint32 auction_type = 2;
-    string nft_id = 3;
-    uint64 start_time = 4 [(gogoproto.moretags) = "yaml:\"start_time\""];
-    uint64 duration = 5 [(gogoproto.moretags) = "yaml:\"duration\""];
-    cosmos.base.v1beta1.Coin min_amount = 6 [(gogoproto.moretags) = "yaml:\"min_amount\"", (gogoproto.nullable) = false];
-    string owner = 7;
-    uint32 limit = 8;
+    AuctionType auction_type = 2;
+    string nft_denom_id = 3;
```

Comment:
What do you think to replace it as we've done with the fantoken?

For example:
Denom => `nft{HASH}`
OldDenom => Collection

Reply:
of course this question should be moved in the nft module

Reply:
Yeah, this discussion belongs to the nft module, not auction module.

Reply:
Can we handle this in another PR? @angelorc 
Since this is related with nft module changes, not auction module.